contribs/gmf/examples/style/mobileapp-vars.less

```diff
@@ -1,8 +1,13 @@
+@app-margin: 13px;
```

Comment:
Let's remove the geolocation button for now, and re-add it later when functional. I'd really prefer the zoom and geolocation buttons to be displayed using the same classes.